cluster/images/hyperkube/Makefile

```diff
@@ -1,8 +1,8 @@
 # build the hyperkube image.
 
-VERSION=v1.1.2
-ARCH=amd64
-BASEIMAGE=debian:jessie
+VERSION?=v1.1.2
```

Comment:
Ref: http://www.gnu.org/software/make/manual/make.html
> There is another assignment operator for variables, ‘?=’. This is called a conditional variable assignment operator, because it only has an effect if the variable is not yet defined. This statement:

> FOO ?= bar
> is exactly equivalent to this (see The origin Function):
> ifeq ($(origin FOO), undefined)
>  FOO = bar
> endif

However, I may remove it if it doesn't keep the consistency with other Makefiles